guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc

```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
 
 ifeval::["{build}" == "satellite"]
 [id="editing-the-system-purpose-of-a-host"]
-== Editing the System Purpose of a Host
+== Editing the System Purpose of Hosts
```

Comment:
Overall I'm not sure why do we replace the procedure for editing a single host, it is still valid, sometimes even  preferable. Unlike the batch edit process, when editing syspurpose attributes in content host details we are able to see what we are changing (i.e. the original values of the attributes)

Reply:
Maybe also worth saying that hosts need to be registered via the subscription manager, In the single host procedure, if the sub-man status of the host is not ok, you are not even able to proceed to the host details to change syspurpose, but with bulk procedure nothing prevents you from trying to update even the non-ok hosts, obviously that results in errors